src/v/redpanda/application.cc

```diff
@@ -2197,6 +2199,30 @@ void application::wire_up_and_start_crypto_services() {
           }
       })
       .get();
+
+    auto sanitized_name = prometheus_sanitize::metrics_name("application");
+
+    _metrics.add_group(
+      sanitized_name,
+      {sm::make_gauge(
+        "fips_mode",
+        [fips_mode_flag] { return static_cast<unsigned int>(fips_mode_flag); },
+        sm::description("Redpanda FIPS Mode State"))});
```

Comment:
Is the internal metric intentionally not aggregated by the shard label?

nit: should the metric be created once as a local variable and copied into the two `.add_group` calls?
```
auto fips_mode_metric = sm::make_gauge(
  "fips_mode",
  [fips_mode_flag] { return static_cast<unsigned int>(fips_mode_flag); },
  sm::description("Redpanda FIPS Mode State"));

_metrics.add_group(sanitized_name, {fips_mode_metric});
_public_metrics.local().groups.add_group(
  sanitized_name, {fips_mode_metric.aggregate({sm::shard_label})});
```

Reply:
> Is the internal metric intentionally not aggregated by the shard label?

As far as I can tell, yes?